ts/features/itwallet/common/utils/itwTypesUtils.ts

```diff
@@ -103,3 +103,12 @@ export type ItwCredentialStatus =
   | "expiring"
   | "expired"
   | ItwJwtCredentialStatus;
+
+// TODO: This will be imported from io-react-native-wallet, when the type will be available
+// Remote presentation QR code data
+export type ItwRemoteQRCodePayload = {
+  clientId: string;
+  requestUri: string;
+  state?: string;
+  requestUriMethod?: string;
+};
```

Comment:
Can we move this under the remote feature folder?
Also `state`, as per the documentation, should be required.